src/utilities/capitalize/index.ts

```diff
@@ -0,0 +1,46 @@
+export const PREPOSITIONS = [
+  'a',
+  'com',
+  'da',
+  'das',
+  'de',
+  'do',
+  'dos',
+  'e',
+  'em',
+  'na',
+  'nas',
+  'no',
+  'nos',
+  'o',
+  'por',
+  'sem',
+];
+
+export function capitalize(value: string, ignoredWords: string[] = []): string {
```

Comment:
Maybe in the future, we will need to add other configs like upperCaseWords (for example `CPF`,` CNPJ` etc), lowerCaseWords, and maybe a transform function to give dev the power to convert any way they want (a really don't know if we need that). So because of that, is better to replace de second parameters to an `object` to give us more flexibility to add new features.

What do you think about that?

Reply:
+1 for replacing the parameter `ignoredWords` to a transform function. :-)

Reply:
I think `upperCaseWords` and `lowerCaseWords` are more common to use than a `transform` function...

Reply:
I'm going to send new changes ...